docs/components/DocumentationSidebar.tsx

```diff
@@ -35,36 +35,42 @@ type SidebarNodeProps = Pick<SidebarProps, 'url' | 'asPath'> & {
   parentRoute?: NavigationRoute;
 };
 
+const renderTypes: Record<NavigationType, React.ComponentType<SidebarNodeProps> | null> = {
+  section: DocumentationSidebarSection,
+  group: DocumentationSidebarGroup,
+  page: null, // Pages are rendered inside groups and should not be rendered directly
```

Comment:
To quickly reiterate the differences:
- **Section** - is a parent node, rendering a collapsible of **groups** inside `section.children`  (we can make it contain pages without requiring group too)
- **Group** - is a parent node, rendering a fixed list of pages inside `group.children`.